src/schemas/Farm.ts

```
@@ -0,0 +1,10 @@
+import * as z from "zod/v4";
+
+const farmSchema = z.object({
```

ESLint

[error] 'farmSchema' is assigned a value but only used as a type.

(@typescript-eslint/no-unused-vars)